build.sbt

```
@@ -58,145 +54,61 @@ val coreDeps = Seq(
-      )
-    ),
-    developers := List(
-      Developer(
-        "ChristopherDavenport",
-        "Christopher Davenport",
-        "chris@christopherdavenport.tech",
-        url("https://github.com/ChristopherDavenport")
-      ),
-      Developer(
-        "amesgen",
-        "Alexander Esgen",
-        "amesgen@amesgen.de",
-        url("https://github.com/amesgen")
-      ),
-      Developer(
-        "rossabaker",
-        "Ross A. Baker",
-        "ross@rossabaker.com",
-        url("https://github.com/rossabaker")
-      )
-    ),
-    githubWorkflowJavaVersions := Seq("11", "17").map(JavaSpec.temurin(_)),
-    githubWorkflowBuild := Seq(
-      WorkflowStep
-        .Sbt(List("scalafmtCheckAll", "scalafmtSbtCheck"), name = Some("Check formatting")),
-      WorkflowStep.Sbt(List("headerCheckAll"), name = Some("Check headers")),
-      WorkflowStep.Sbt(List("Test/compile"), name = Some("Compile")),
-      WorkflowStep
-        .Sbt(List("core/mimaReportBinaryIssues"), name = Some("Check binary compatibility")),
-      WorkflowStep.Sbt(
-        List("unusedCompileDependenciesTest", "undeclaredCompileDependenciesTest"),
-        name = Some("Check unused and undeclared compile dependencies")
-      ),
-      WorkflowStep.Sbt(List("test"), name = Some("Run tests")),
-      WorkflowStep.Sbt(List("doc"), name = Some("Build docs"))
-    ),
-    githubWorkflowPublishPostamble := Seq(
-      WorkflowStep.Run(
-        List("""
-             |eval "$(ssh-agent -s)"
-             |echo "$SSH_PRIVATE_KEY" | ssh-add -
-             |git config --global user.name "GitHub Actions CI"
-             |git config --global user.email "ghactions@invalid"
-             |sbt ++${{ matrix.scala }} docs/ghpagesPushSite
-             """.stripMargin),
-        name = Some("Publish docs"),
-        env = Map(
-          "SSH_PRIVATE_KEY" -> "${{ secrets.SSH_PRIVATE_KEY }}",
-          "SBT_GHPAGES_COMMIT_MESSAGE" -> "Updated site: sha=${{ github.sha }} build=${{ github.run_id }}"
-        ),
-        cond = Some("github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v')")
-      )
-    ),
-    githubWorkflowPublishTargetBranches := Seq(
-      RefPredicate.Equals(Ref.Branch("main")),
-      RefPredicate.StartsWith(Ref.Branch("series/")),
-      RefPredicate.StartsWith(Ref.Tag("v"))
-    )
+ThisBuild / crossScalaVersions := Seq("2.12.15", "2.13.7", "3.1.0")
+ThisBuild / scalaVersion := (ThisBuild / crossScalaVersions).value.head
+ThisBuild / tlBaseVersion := "0.7"
+ThisBuild / startYear := Some(2021)
+ThisBuild / developers := List(
+  Developer(
+    "ChristopherDavenport",
+    "Christopher Davenport",
+    "chris@christopherdavenport.tech",
+    url("https://github.com/ChristopherDavenport")
+  ),
+  Developer(
+    "amesgen",
+    "Alexander Esgen",
+    "amesgen@amesgen.de",
+    url("https://github.com/amesgen")
+  ),
+  Developer(
+    "rossabaker",
+    "Ross A. Baker",
+    "ross@rossabaker.com",
+    url("https://github.com/rossabaker")
```

```suggestion
  tlGitHubDev("ChristopherDavenport", "Christopher Davenport"),
  tlGitHubDev("amesgen", "Alexander Esgen"),
  tlGitHubDev("rossabaker", "Ross A. Baker")
```
Feel free to also add yourself if you want ;)

---

Sry, ust noted that I missed one closing `)` here...